### DISCUSSION TITLE
line filter

### DISCUSSION THREAD

**Discussion by Criscia68** [WinMerge/winmerge#1126] [General]
Is it possible in a line to exclude "from" a character to the end of the line? 

    I_4_plus_not_overtravelled              I     682                     ;;;PROFIBUS

starting from ; do not check the differences.

**Comment by sdottaka** [comment]
Please use Substitution filters or PrediffLineFIlter.sct plugin

![image](https://user-images.githubusercontent.com/98126/148949148-cd02cada-603d-48eb-add1-ec7f72284784.png)

**Comment by Criscia68** [comment]
![image](https://user-images.githubusercontent.com/85893336/148952897-f87e3a76-4d00-4456-8500-8acd20973689.png)

![image](https://user-images.githubusercontent.com/85893336/148952658-d83db2e7-ab7c-4a8d-b522-0573d60ac1e0.png)

![image](https://user-images.githubusercontent.com/85893336/148952836-cc7a2974-ef55-46cc-bfb8-c80a3a85d1cb.png)

![image](https://user-images.githubusercontent.com/85893336/148952588-1ee35108-5347-44f4-8f4f-137fce69bf69.png)

This filter appears to be inactive line by line.
